### DISCUSSION TITLE
OnBlur doesn't work after form reset?

### DISCUSSION THREAD

**Discussion by Eerkz** [react-hook-form/react-hook-form#8165] [Q&A]
Whenever i reset my form, onBlur doesn't seem to work anymore. Here's my settings for reset:

```
  const resetForm = () => {
    reset(defaultValues, {
      keepErrors: false,
      keepDirty: false,
    })}
```

**Comment by Moshyfawn** [comment]
What do you mean by `onBlur`?
```js
<input register("fieldName", { onBlur: () => console.log("BLUR") }) />
```
or
```js
useForm({ mode: "onBlur" })
```

Care to provide a reproduction codesanbox?

**Reply by Eerkz** [reply]
I meant validation mode onBlur so the latter.

**Reply by bluebill1049** [reply]
provide a codesandbox.

**Comment by Moshyfawn** [comment]
Yea, a reproduction codesanbox would be handy. 

One thing: do you have [the RHF `Devtool` component](https://github.com/react-hook-form/devtools) in the form by chance?..
